.github/workflows/swiftshader.yml

```diff
@@ -0,0 +1,40 @@
+name: SwiftShader
+
+on: push
```

Comment:
This action will trigger whenever any branch is pushed. I don't think we want that. Did you mean to replace this with the `workflow_dispatch` below?

Reply:
I actually wasn't sure, I wanted to ask you guys about that.  I was leaning towards the manual trigger via workflow_dispatch.

Reply:
I'm fine with a manual trigger, but it's a bit tricky to set up. You have to get a GitHub personal access token and then make a POST request to a specific GitHub API: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#repository_dispatch